Sources/PointFree/TODO.swift

```diff
@@ -49,10 +49,14 @@ public func requireSome<A>(
 
 // TODO: Move to PreludeFoundation?
 
+private let sessionConfig = URLSessionConfiguration.default
+  |> \.timeoutIntervalForRequest .~ 30
+  |> \.timeoutIntervalForResource .~ 30
```

Comment:
maybe this should be 25 since Heroku's timeout is 30, and so we prob won't have enough time to recover.

Reply:
yeah let’s do it